app/livechat/client/views/app/tabbar/visitorEdit.html

```diff
@@ -31,16 +31,18 @@ <h3>{{username}}</h3>
 					</label>
 				</div>
 
-				{{#each visitorCustomFields}}
-					<div class="rc-input rc-form-group rc-form-group--small">
-						<label class="rc-input__label">
-							<div class="rc-input__title">{{label}}</div>
-							<div class="rc-input__wrapper">
-								<input class="rc-input__element" type="text" name="{{name}}" autocomplete="off" data-visitorLivechatData="true" value="{{value}}">
-							</div>
-						</label>
-					</div>
-				{{/each}}
+				{{#if canViewCustomFields }}
```

Comment:
We need to apply the same here:
https://github.com/RocketChat/Rocket.Chat/blob/develop/app/livechat/client/views/app/tabbar/visitorEdit.html#L95